cmd/util/cmd/epochs/cmd/deploy.go

```diff
@@ -256,14 +257,18 @@ func getDeployEpochTransactionText(snapshot *inmem.Snapshot) []byte {
 	chainID := head.ChainID
 	systemContracts := systemcontracts.SystemContractsForChain(chainID)
 
+	env := coreContractstemplates.Environment{
```

Comment:
can we just use `systemContracts.AsTemplateEnv()` everywhere?

Reply:
That would require a circular dependency because `core-contracts/lib/go/templates` uses `templates.Environment` to replace the address placeholders, so I'd prefer not to do that